package.json

```diff
@@ -63,6 +63,7 @@
     "micromatch": "^4.0.4",
     "mlh-tsd": "^0.14.1",
     "mock-fs": "^4.4.1",
+    "netlify-plugin-cache": "^1.0.3",
```

Comment:
should be in `website/package.json`, no?

Reply:
Using netlify with monorepos is sometimes a bit confusing. I find that putting netlify.toml and netlify plugins at the root is the simplest solution that works fine when monorepo only contains a single website

I just reverted the commit you already merged/reverted btw